samples/mycompany.sh

```diff
@@ -27,6 +27,15 @@ An application is a piece of software that
 can be deployed independently in the information system to offer an identified high level service
 '
 
+curl --request PUT \
+  --url $API_BASE_URL/v1/contents/help-what-is-a-repository/en-GB \
+  --header 'content-type: text/plain' \
+  --data 'Code repository
+------
+
+Code of application is stored in a __code repository__ (often git)
+'
+
 curl --request PUT \
   --url $API_BASE_URL/v1/environments/prodfr \
   --header 'content-type: application/json' \
```

Comment:
Should we put default values in the DB migration scripts ?

Reply:
yes on integration side